documentation_test.go

```diff
@@ -69,11 +70,15 @@ func ExampleSynopsis() { //nolint: govet
 	// 4
 }
 
-func ExampleConsole() { //nolint: govet
+func ExampleConsole() {
```

Comment:
Feels like we should update this to use the new WithConsole?

Reply:
I would suggest to leave this as is and rename plus add a comment that this offers even more control over the console (i.e. add the unimplemented functions of the `Console` interface). It also shows how to extend js objects in the first place. Had I known this existed I would probably not have created this PR at all.

That said, if you agree I'd work on the examples to finalise this PR?

Reply:
Sounds good.